### ISSUE TITLE
Update project license to use spdx identifier for the "Creative Commons Attribution 4.0 International" license.

### ISSUE THREAD

**Issue by retslig** [mono/SkiaSharp#621]
I was hoping the project owner would consider updating the project license to use spdx identifier for the "Creative Commons Attribution 4.0 International" license.  Your license in github -> https://github.com/mono/SkiaSharp/blob/master/LICENSE
This is no change to the license at all, but then allows anyone using the github license API to get an exact match of the licensed used and not have to use any string magic to come up with your license.

If your unfamiliar with SPDX see spdx.org. And your particular license I am guessing would be https://spdx.org/licenses/CC-BY-4.0.html.

Pretty sure this can be done in your settings somewhere.  Here is an example of your API json and your 
spdx_id is null.

{
  "name": "LICENSE",
  "path": "LICENSE",
  "sha": "e056e7c3ec54533985dfb3a4c6ef70dab8ca2e42",
  "size": 18650,
  "url": "https://api.github.com/repos/mono/SkiaSharp/contents/LICENSE?ref=master",
  "html_url": "https://github.com/mono/SkiaSharp/blob/master/LICENSE",
  "git_url": "https://api.github.com/repos/mono/SkiaSharp/git/blobs/e056e7c3ec54533985dfb3a4c6ef70dab8ca2e42",
  "download_url": "https://raw.githubusercontent.com/mono/SkiaSharp/master/LICENSE",
  "type": "file",
  "content": "removed as this was very long"
  "encoding": "base64",
  "_links": {
  "self": "https://api.github.com/repos/mono/SkiaSharp/contents/LICENSE?ref=master",
  "git": "https://api.github.com/repos/mono/SkiaSharp/git/blobs/e056e7c3ec54533985dfb3a4c6ef70dab8ca2e42",
  "html": "https://github.com/mono/SkiaSharp/blob/master/LICENSE"
  },
  "license": {
  "key": "other",
  "name": "Other",
  "spdx_id": null,
  "url": null,
  "node_id": "MDc6TGljZW5zZTA="
  }
}

I also noticed this URL in your nuget package "https://go.microsoft.com/fwlink/?linkid=868514" which then redirects you to "https://github.com/mono/SkiaSharp/blob/master/LICENSE.md" which is a Xamarin/Microsoft license...  So not sure what that is about either...
2 different licenses?

**Comment by mattleibow** [comment]
Thanks for reporting this and I had a look. It appears that we use the CC license for just the docs, and the process we used to add the license for the docs did not understand that docs could be in a sub-folder. I have corrected the files and now they should be in the correct places.

The license we use for SkiaSharp is really MIT. The API documentation content are under the CC. And, skia and it's native files are all under their respective licenses, all of which can be found in this big file: https://github.com/mono/SkiaSharp/blob/master/External-Dependency-Info.txt

This repository contains those three main areas:
 - skia (under the Google license) and its dependencies (under their own licenses)
 - SkiaSharp (under the Microsoft / MIT license)
 - the docs (under the Creative Commons)
